README.md

```diff
@@ -2,7 +2,9 @@
 # Hello-Rails
 This is a simple project just to learn how to build an only api rails app.
 
-A simple hello world app. This is just a work around. In order to learn more about how to build a react and rails app This app is the backend.
+
+A simple hello world app. This is just a work around. In order to learn more about how to build a react and rails appThis app is the backend.
+
```

Comment:
- [ ] In the README file, Please mention the port as well in the steps where we are running the rails server.


- [ ] In the README file, Please link the Frontend application and vice versa. Currently, you have put the frontend app link in the Pull Request but you should add it to the README.